lua/mason-null-ls/api/command.lua

```diff
@@ -106,13 +106,15 @@ local NullLsInstall = a.scope(function(sources)
 	end
 end)
 
-vim.api.nvim_create_user_command('NullLsInstall', function(opts)
-	NullLsInstall(opts.fargs)
-end, {
-	desc = 'Install one or more Null-ls sources.',
-	nargs = '*',
-	complete = 'custom,v:lua.mason_null_ls_completion.available_source_completion',
-})
+for _, command in ipairs({ 'NullLsInstall', 'NoneLsInstall' }) do
+	vim.api.nvim_create_user_command(command, function(opts)
+		NullLsInstall(opts.fargs)
+	end, {
+		desc = 'Install one or more Null-ls sources.',
```

Comment:
As null-ls has gone into the shadow realm maybe the names could be switched in help texts?

```suggestion
		desc = 'Install one or more none-ls sources.',
```